### DISCUSSION TITLE
Firefly III does not provide me the right charset  in the exported transaction file openning with excel application. All Chinese characters are showing weird

### DISCUSSION THREAD

**Discussion by zhanghengtian** [firefly-iii/firefly-iii#5739] [Firefly III] [answered]
Firefly III does not provide me the right charset  in the exported transaction file all Chinese characters are showing weird . I have already convert the file as UTF-8 coded and it displays fine in the source file and fireflyiii, but showing weirod in the exported file when opening with Excel. 

Is this the configuration issue occured by Excel or somehow the database charset setting is incorrect?

**Comment by JC5** [comment, marked as answer]
Please double-check if Excel opens and converts the CSV file as UTF-8.

**Reply by zhanghengtian** [reply]
thanks for the quick respond.
I googled this problem is fixed. 
I shall use [open from csv file] command under data tab in Excel and it showing fine.
